building/build-debs/homeworld-admin-tools/debian/rules

```diff
@@ -17,11 +17,11 @@ include /usr/share/dpkg/default.mk
 #export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
 
 override_dh_auto_build:
-	VERSION="${DEB_VERSION_UPSTREAM}" ./inner-build.sh
+	DEB_VERSION="${DEB_VERSION_UPSTREAM}" ./inner-build.sh
```

Comment:
Is it necessary to rename this? It breaks consistency with the other debian/rules files.

Reply:
Not necessary. It just makes the variable naming in inner-build.sh nicer. Do you want me to change it back?

Reply:
Yeah -- if you want to submit a separate change to improve the naming, that'd be good, but perhaps not in this PR.